listView.php

```diff
@@ -157,11 +157,12 @@
 echo "<div class='row'>";
 foreach ($lists as $list){
   //This code is run FOR EACH list that exists.
+  //Each list name can be clicked to access the filtering functionality(UX needs to be improved a little)
 	echo "
 	<div class='col-xs-3'>
 	<div class='card'>
 	<div class='card-header'>
-    <div class ='card-Title'>
+    <div class ='card-Title' onclick='dynamicModal(".$list['listID'].")' data-toggle='modal' data-target='#filterList'>
```

Comment:
maybe add some CSS or a icon to represent filtering as the user would not know to click on the title of the list to view the filtering options.

Reply:
Thanks for this! I have modified the css to change the mouse icon to a pointer when hovering over the list name, so that the user will now notice that it's click-able.